cloud_governance/policy/helpers/aws/policy/zombie_cluster_operations.py

```diff
@@ -0,0 +1,186 @@
+                                     tags_name: str,
+                                     create_date: str) -> dict:
+        """
+        This method process the zombie_cluster_resources and delete the resources
+        :param create_date:
+        :param resource_type:
+        :param zombie_cluster_resources:
+        :param resource_id_key:
+        :param tags_name:
+        :return:
+        """
+        zombie_cluster_resources_response = {}
+        for zombie_cluster_tag, resources in zombie_cluster_resources.items():
+            resource_ids = []
+            tags = []
+            found_create_date = None
+            for resource in resources:
+                resource_ids.append(resource[resource_id_key])
+                if not found_create_date:
+                    if not create_date:
+                        found_create_date = resource.get(create_date)
+                if not tags:
+                    tags = resource.get(tags_name, [])
+            cleanup_days = self.get_clean_up_days_count(tags=tags, tag_name='CleanUpDays')
+            policy_response = self.zombie_cluster_verify_and_delete_resource(resource_ids,
+                                                                             tags=tags,
+                                                                             clean_up_days=cleanup_days,
+                                                                             resource_type=resource_type)
+            zombie_cluster_resources_response[zombie_cluster_tag] = {
+                'ResourceIds': resource_ids,
+                'DaysCount': self.get_clean_up_days_count(tags, tag_name='CleanUpDays'),
+                'ClenUpResult': policy_response.deleted,
+                'Message': policy_response.message,
+                'Error': policy_response.error,
+                'User': self.get_tag_name_from_tags(tags=tags, tag_name='User'),
+                'ZombieClusterTag': zombie_cluster_tag,
+                'CreateDate': str(found_create_date if found_create_date else ""),
+            }
+            if not policy_response.deleted:
+                self.update_resource_tags(tags=tags,
+                                          resource_ids=resource_ids,
+                                          resource_type=resource_type,
+                                          cleanup_days=self.get_clean_up_days_count(tags),
+                                          tag_name='CleanUpDays')
+        return zombie_cluster_resources_response
+
+    def _delete_resource(self, resource_id: list):
+        """
+        This method deletes the resource
+        :param resource_id:
+        :return:
+        """
+        resource_ids = resource_id
+        if resource_ids:
+            resource_id = resource_ids[0]
+            try:
+                if 'vol' in resource_id:
+                    self._ec2_operations.delete_volumes(resource_ids=resource_ids)
+                elif 'sg' in resource_id:
```

Comment:
Single method to delete all the resources of zombie_cluster, Instead of splitting to multiple methods.